bats/fb-content-katello.bats

```diff
@@ -140,12 +140,21 @@ EOF
   subscription-manager repos --enable="${ORGANIZATION_LABEL}_${PRODUCT_LABEL}_${YUM_REPOSITORY_LABEL}" | grep -q "is enabled for this system"
 }
 
-@test "install katello-agent" {
-  tPackageInstall katello-agent && tPackageExists katello-agent
+@test "install katello-host-tools" {
+  tPackageInstall katello-host-tools && tPackageExists katello-host-tools
+}
+
+@test "install package locally" {
+  run yum -y remove walrus
```

Comment:
I don't know actually, but I don't consider this an assertion that we are testing, more of test cleanup so my preference is to use run in those situations. SInce the katello-host-tools has been merged to packaging, I'm going to merge this in order for it to be tested.